modules/ROOT/pages/setup-ansible-playbooks.adoc

```diff
@@ -72,35 +81,28 @@ The jumphost will serve three purposes:
 [TIP]
 ====
 To simplify accessing the jumphost and {product-proxy} instances from your machine, create a xref:deployment-infrastructure.adoc#_connecting_to_the_zdm_infrastructure_from_an_external_machine[custom SSH configuration file].
-The following steps will assume that this file exists.
+The following steps assume that this file exists.
```

Comment:
The link above is now at
xref:deployment-infrastructure.adoc#connect-to-zdm-proxy-infrastructure-from-an-external-machine